open-api.yaml

```diff
@@ -1215,40 +1232,47 @@ paths:
               properties:
                 q:
                   type: string
-                  description: Query string _(mandatory)_.
+                  description: Query string.
+                  default: '""'
+                  example: '"Back to the future"'
                 attributesToRetrieve:
                   type: array
-                  description: Attributes to display in the returned documents.
+                  description: 'Array of attributes whose fields will be present in the returned documents. Defaults to the [displayedAttributes list](/reference/features/settings.html#displayed-attributes) which contains by default all attributes found in the documents.'
```

Comment:
I see this is written in the context of `post:` whereas the previous `attributesToRetrieve` was not in the context of `get:`

Same for all array parameters

Reply:
I distinguised parameters for POST and GET since they doesn't share the same syntax.